docs/standard/garbage-collection/fundamentals.md

```diff
@@ -150,57 +150,45 @@ Before a garbage collection starts, all managed threads are suspended except for
 
-- Workstation garbage collection, which is for all client workstations and stand-alone PCs. This is the default setting for the [\<gcServer> element](../../../docs/framework/configure-apps/file-schema/runtime/gcserver-element.md) in the runtime configuration schema.
-
-  Workstation garbage collection can be concurrent or non-concurrent. Concurrent garbage collection enables managed threads to continue operations during a garbage collection.
-
-  Starting with the .NET Framework 4, background garbage collection replaces concurrent garbage collection.
-
-- Server garbage collection, which is intended for server applications that need high throughput and scalability. Server garbage collection can be non-concurrent or background.
-
-The following illustration shows the dedicated threads that perform the garbage collection on a server.
+- Workstation garbage collection, which is for all client workstations and stand-alone PCs. This is the default setting for the [gcServer element](../../../docs/framework/configure-apps/file-schema/runtime/gcserver-element.md) in the runtime configuration schema.
 
-![Server Garbage Collection Threads](../../../docs/standard/garbage-collection/media/gc-server.png "Server Garbage Collection Threads")
+  Workstation garbage collection can be concurrent or non-concurrent. Concurrent garbage collection enables managed threads to continue operations during a garbage collection. [Background garbage collection](#background-garbage-collection) replaces [concurrent garbage collection](#concurrent-garbage-collection) in .NET Framework 4 and later versions.
 
-### Configuring garbage collection
+- Server garbage collection, which is intended for server applications that need high throughput and scalability.
 
-You can use the [\<gcServer> element](../../../docs/framework/configure-apps/file-schema/runtime/gcserver-element.md) of the runtime configuration schema to specify the type of garbage collection you want the CLR to perform. When this element's `enabled` attribute is set to `false` (the default), the CLR performs workstation garbage collection. When you set the `enabled` attribute to `true`, the CLR performs server garbage collection.
+  - In .NET Core, server garbage collection can be non-concurrent or background.
 
-Concurrent garbage collection is specified with the [\<gcConcurrent> element](../../../docs/framework/configure-apps/file-schema/runtime/gcconcurrent-element.md) of the runtime configuration schema. The default setting is `enabled`. This setting controls both concurrent and background garbage collection.
+  - In .NET Framework 4.5 and later versions, server garbage collection can be non-concurrent or background (background garbage collection replaces concurrent garbage collection). In .NET Framework 4 and previous versions, server garbage collection can be concurrent or non-concurrent.
```

Comment:
>server garbage collection can be concurrent or non-concurrent [](start = 223, length = 61)

Server GC was never concurrent before 4.5. concurrent Server GC (ie, Background Server GC) was only enabled in 4.5